tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "test/**/*.ts"
   ],
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "test/fixtures"
```

Comment:
Another way doing this would have been to drop the `test/**/*.ts` from the `includes` above, but this is fine.